src/components/Pressable/GenericPressable/BaseGenericPressable.tsx

```diff
@@ -135,39 +128,37 @@ const GenericPressable = forwardRef((props, ref) => {
             ref={ref}
             onPress={!isDisabled ? singleExecution(onPressHandler) : undefined}
             onLongPress={!isDisabled && onLongPress ? onLongPressHandler : undefined}
-            onKeyPress={!isDisabled ? onKeyPressHandler : undefined}
             onKeyDown={!isDisabled ? onKeyDown : undefined}
             onPressIn={!isDisabled ? onPressIn : undefined}
             onPressOut={!isDisabled ? onPressOut : undefined}
             style={(state) => [
-                getCursorStyle(shouldUseDisabledCursor, [props.accessibilityRole, props.role].includes('text')),
-                StyleUtils.parseStyleFromFunction(props.style, state),
-                isScreenReaderActive && StyleUtils.parseStyleFromFunction(props.screenReaderActiveStyle, state),
-                state.focused && StyleUtils.parseStyleFromFunction(props.focusStyle, state),
-                state.hovered && StyleUtils.parseStyleFromFunction(props.hoverStyle, state),
-                state.pressed && StyleUtils.parseStyleFromFunction(props.pressStyle, state),
-                isDisabled && [...StyleUtils.parseStyleFromFunction(props.disabledStyle, state), styles.noSelect],
+                getCursorStyle(shouldUseDisabledCursor, [rest.accessibilityRole, rest.role].includes('text')),
+                StyleUtils.parseStyleFromFunction(style, state),
+                isScreenReaderActive && StyleUtils.parseStyleFromFunction(screenReaderActiveStyle, state),
+                state.focused && StyleUtils.parseStyleFromFunction(focusStyle, state),
+                state.hovered && StyleUtils.parseStyleFromFunction(hoverStyle, state),
+                state.pressed && StyleUtils.parseStyleFromFunction(pressStyle, state),
+                isDisabled && [StyleUtils.parseStyleFromFunction(disabledStyle, state), styles.noSelect],
             ]}
             // accessibility props
             accessibilityState={{
                 disabled: isDisabled,
-                ...props.accessibilityState,
+                ...rest.accessibilityState,
             }}
             aria-disabled={isDisabled}
-            aria-keyshortcuts={keyboardShortcut && `${keyboardShortcut.modifiers}+${keyboardShortcut.shortcutKey}`}
+            aria-keyshortcuts={keyboardShortcut && `${keyboardShortcut.modifiers.join('')}+${keyboardShortcut.shortcutKey}`}
```

Comment:
why this logic change?

Reply:
Due to this: https://github.com/Expensify/App/pull/30777#discussion_r1381414518